.github/workflows/ci.yml

```diff
@@ -15,14 +15,19 @@ jobs:
         with:
           python-version: 3.9
       - name: install dependencies
-        run: python -m pip install flake8
-      - name: lint with flake8
+        run: python -m pip install -r requirements-dev.txt -e .
+      - name: Check lint with flake8
         run: flake8 --statistics easypost/ tests/ examples/
+      - name: Check formatting with Black
+        run: black easypost --check
+      - name: Check formatting with iSort
+        run: isort easypost --check-only
```

Comment:
I think it's worth running these on the tests and examples directories as well

Reply:
Couldn't agree more, overlooked those - thanks!